sonar-ws/src/main/java/org/sonarqube/ws/client/setting/SetRequest.java

```diff
@@ -75,11 +85,16 @@ public Builder setKey(String key) {
       return this;
     }
 
-    public Builder setValue(String value) {
+    public Builder setValue(@Nullable String value) {
       this.value = value;
       return this;
     }
 
+    public Builder setValues(@Nullable List<String> values) {
```

Comment:
as we accept null value in the output, the `SetRequest.getValues()` can return null